README.md

```diff
@@ -16,3 +12,13 @@ Community Meeting summaries/minutes and logs are posted to https://github.com/an
 4. At the meeting, we look at the list of open issues, and issues with `next_meeting` get first priority.
     * Currently we assume that the list of open issues won't be too long, and that not too many will have a `next_meeting` label, so we don't need a fully defined process on how to select topics for the meeting.
     * If we find out that this assumption isn't good, we can re-evaluate/discuss this in a meeting.
+
+## Community meetings
+
+Some of the topics can be additionally discussed in the weekly Community Meetings though final decisions are always made by **asynchronous** voting in the topics.
+
+See the Community Working Group meeting [schedule](https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays) and join us in any of the following bridged channels:
+* on [Matrix](https://docs.ansible.com/ansible/latest/community/communication.html#ansible-community-on-matrix): `#community:ansible.im`
```

Comment:
`ansible.com` for links (not `.im)

Also, we should give a Matrix.to link, ie `https://matrix.to/#/#community:ansible.com`

Reply:
fixed, thanks, ptal